DateTime/index.spec.ts

```diff
@@ -74,6 +74,12 @@ describe("DateTime", () => {
 	] as const)("create(%d, %s) == %s", (value, resolution, expected) =>
 		expect(isoly.DateTime.create(value, resolution)).toBe(expected)
 	)
+	it.each([
+		["2023-10-31T11:23:40.8Z", "2023-10-31T11:23:40.800Z"],
+		["2023-10-31T11:23:40.81Z", "2023-10-31T11:23:40.810Z"],
+		["2023-10-31T11:23:40Z", "2023-10-31T11:23:40Z"],
+		["2023-10-31T11:23:40.000Z", "2023-10-31T11:23:40.000Z"],
+	])("normalize(%s) == %s", (value, expected) => expect(isoly.DateTime.normalize(value)).toEqual(expected))
```

Comment:
Add test cases for several decimal digits and without timezone
```ts
["2023-10-31T11:23:40.123678978Z", "2023-10-31T11:23:40.123Z"],
["2023-10-31T11:23:40.223678978", "2023-10-31T11:23:40.223"],
```

Reply:
Added the tests above.